src/main/java/Homework_number5.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Homework_number5 {
+
+    public static void main(String[] args) {
+
+        System.out.println(findSymbolOccurance("abbccc", 'c'));
+        System.out.println(findWordPosition("aaabbmmm", "bbb"));
+        System.out.println(reverseString("Hello"));
+        System.out.println(isPalindrome("abccba"));
+
+        playGame();
+
+    }
+
+    /* Створити метод findSymbolOccurance. Метод приймає як параметр рядок і символ.
+    Необхідно обчислити, скільки разів символ зустрічається в переданому рядку
+    і повернути це числове значення. */
+    public static int findSymbolOccurance(String inputString, char inputChar) {
```

Comment:
The input parameters of a method should be validated. For now, would be great if you check if 'inputString' is not null or NullPointerException might be thrown otherwise.